src/loggerMiddleware.lua

```diff
@@ -1,4 +1,3 @@
---!strict
```

Comment:
Any way we could make this analyze instead of making it less strict?

Reply:
I don't mind re-adding the explicit strict mode here if you want. The reason I removed it is because I was hitting useless type errors in my integration test wherever we directly used middleware. It's a minor inconvenience but it will require an explicit cast in wherever we consume them.

Reply:
I didn't bother chasing it further because this doesn't seem like it would be a significant source of bugs. AFAICT this is just a nice to have.